scripts/lib/CIME/case/check_input_data.py

```diff
@@ -137,7 +137,8 @@ def check_all_input_data(self, protocol=None, address=None, input_data_root=None
             inputdata = Inputdata()
             protocol = "svn"
             success = False
-            while not success and protocol is not None:
+            # download and merge all available chksum files.
+            while protocol is not None:
```

Comment:
Since you no longer use the `success` value returned from the call to `_download_checksum_file`, you should probably change that to `_` to signal that the return value isn't used.

Reply:
fixed

Reply:
Thanks. I'll wait to approve until you answer my more general question.